model-optimizer/mo/utils/versions_checker.py

```diff
@@ -146,6 +187,21 @@ def version_check(name, installed_v, required_v, sign, not_satisfied_v, exit_cod
     return exit_code
 
 
+def get_environment_variables():
+    """
+    Checks if installed modules versions satisfy required versions in requirements file
+    :return: a dictionary of environment variables
+    """
+    env_params = dict()
+    exec("import sys")
```

Comment:
why cant we do it without exec? sys module is standard and there is no way it is not installed

Reply:
Fixed. Thanks